docs/modules/contributions/pages/contributions.adoc

```diff
@@ -12,6 +21,405 @@ To install a jMonkeyEngine SDK plugin, go to `menu:Tools[Plugins>Available Plugi
+
+|{url-wiki}/networking/networking.html[SpiderMonkey]
+|Multi-player networking
+|{url-core}[Core developers]
+|{url-bintray}/jmonkeyengine/org.jmonkeyengine/jme3-networking[org.jmonkeyengine:jme3-networking]
+|{url-enginelib}/jme3-networking[GitHub]
+
+|{url-wiki}/core/gui/nifty_gui.html[Nifty]
+|Graphical user interface
+|{url-core}[Core developers]
+|{url-bintray}/jmonkeyengine/org.jmonkeyengine/jme3-niftygui[org.jmonkeyengine:jme3-niftygui]
+|{url-enginelib}/jme3-niftygui[GitHub]
+
+|{url-wiki}/documentation.html[jme3-plugins]
+|Import more file formats
+|{url-core}[Core developers]
+|{url-bintray}/jmonkeyengine/org.jmonkeyengine/jme3-plugins[org.jmonkeyengine:jme3-plugins]
+|{url-enginelib}/jme3-plugins[GitHub]
+
+|{url-wiki}/core/terrain/terrain.html[TerraMonkey]
+|Terrain
+|{url-core}[Core developers]
+|{url-bintray}/jmonkeyengine/org.jmonkeyengine/jme3-terrain[org.jmonkeyengine:jme3-terrain]
+|{url-enginelib}/jme3-terrain[GitHub]
+
+|{url-wiki}/core/vr/virtualreality.html[jme3-vr]
+|Virtual reality
+|{url-core}[Core developers]
+|{url-bintray}/jmonkeyengine/org.jmonkeyengine/jme3-vr[org.jmonkeyengine:jme3-vr]
+|{url-enginelib}/jme3-vr[GitHub]
+
+
+|===
+
+=== At other public repositories
+
+The table below lists JME-related libraries available primarily
+from public Maven repositories other than JCenter.
+"GroupID:ArtifactID" entries link to package information;
+follow these links to determine the version ID of the latest release.
+"Name" entries link to relevant documentation, if any.
+
+[cols="20,20,15,35,10",grid="none",options="header"]
+|===
+|Name
+|Purpose
+|Maintainer(s)
+|Maven repository URL +
+ GroupID:ArtifactID
+|Source code
+
+|{url-github}/riccardobl/jme3-bullet-vhacd#readme[V-HACD Collision Shape Factory]
+|Decompose meshes into convex collision shapes
+|{url-forum-user}/RiccardoBlb[RiccardoBlb]
+|\https://jitpack.io +
+ {url-jitpack}/#riccardobl/jme3-bullet-vhacd[com.github.riccardobl:jme3-bullet-vhacd]
+|{url-github}/riccardobl/jme3-bullet-vhacd[GitHub]
+
+|{url-github}/riccardobl/jme-igui#readme[jme-IGUI]
+|Immediate graphical user interface
+|{url-forum-user}/RiccardoBlb[RiccardoBlb]
+|\https://jitpack.io +
+ {url-jitpack}/#riccardobl/jme-igui[com.github.riccardobl:jme-igui]
+|{url-github}/riccardobl/jme-igui[GitHub]
+
+
+|{url-github}/riccardobl/jme-igui#readme[Effekseer Native]
+|Render effects made with Effekseer
+|{url-forum-user}/RiccardoBlb[RiccardoBlb]
+|\https://dl.bintray.com/riccardo/effekseer +
+ {url-bintray}/riccardo/effekseer/jme-effekseerNative[com.jme.effekseer:jme-effekseerNative]
+|{url-github}/riccardobl/jme-effekseerNative[GitHub]
+
+|{url-github}/jmePhonon/jmePhonon#readme[jmePhonon]
+|Steam(R) audio
+|{url-forum-user}/RiccardoBlb[RiccardoBlb]
+|\https://dl.bintray.com/jmephonon/jmePhonon +
+ {url-bintray}/jmephonon/jmePhonon/jmePhonon[com.jme3.phonon:jmePhonon]
+|{url-github}/jmePhonon/jmePhonon[GitHub]
+
+
+|{url-github}/rvandoosselaer/Blocks/wiki[Blocks]
+|Voxel worlds
```

Comment:
Could you change this to 'Customizable voxel engine'?

Reply:
I moved it to the JCenter section and changed the description as you suggested.

Btw, the Blocks wiki still recommends adding the Bintray repository. You may want to update that.